yarn-project/p2p/src/client/p2p_client.test.ts

```diff
@@ -103,8 +105,8 @@ describe('In-Memory P2P Client', () => {
     await client.start();
     expect(client.isReady()).toEqual(true);
 
-    await client.stop();
-    expect(client.isReady()).toEqual(false);
+    // await client.stop();
```

Comment:
Should this be commented out?

Reply:
Oh, I commented out for debugging. Will restore